docs/custom-plugins/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Custom plugins in Redocly
+redirectFrom:
+  - /docs/cli/custom-rules/
+  - /docs/cli/resources/custom-rules/
+  - /docs/cli/resources/custom-plugins/
+---
+
+# Custom plugins
+
+Custom plugins are a powerful way to extend Redocly. Use of custom plugins requires requires in-depth knowledge of the document type (such as OpenAPI) that you are working with, JavaScript coding skills, and the plugin interface.
+Custom plugins are recommended for advanced users who have exhausted the built-in options available, and who can develop JavaScript extensions themselves.
+Each plugin is a JavaScript module which can export custom rules, configuration, preprocessors, decorators or type tree extensions.
```

Comment:
```suggestion
Each plugin is a JavaScript module which can export custom rules, configurations, preprocessors, decorators or type tree extensions.
```

Reply:
So it might be an American/English thing, because from what I can tell configuration can be made plural with an "s" in American English and it is fine. I do not have particularly strong feelings either way though, so your choice.

Reply:
If it  seems off to you, it probably seems wrong to other US English speakers so I'm accepting the suggestion. Happy to hear that either would be valid though!